airbyte-integrations/connectors/source-gong/source_gong/manifest.yaml

```
@@ -80,15 +104,27 @@ definitions:
       requester:
         $ref: "#/definitions/requester"
         http_method: "POST"
-        request_body_json:
-          filter: '{"callFromDate": "{{ config["start_date"] }}"}'
+    incremental_sync:
+      type: CustomIncrementalSync
+      class_name: source_gong.components.IncrementalSingleBodyFilterCursor
+      cursor_field: reviewTime
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S.%f%z"
+      datetime_format: "%Y-%m-%d"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ config['start_date'] }}"
```

```suggestion
        datetime: "{{ config,get('start_date', now().subtract(days=-180) }}"
```
What about adding a default value when users didn't specify the start date? This way it won't introduce a breaking change.

---

you're right, added :white_check_mark: